python/pyspark/sql/pandas/group_ops.py

```diff
@@ -114,12 +157,12 @@ def __init__(self, gd1, gd2):
         self.sql_ctx = gd1.sql_ctx
 
     @since(3.0)
-    def apply(self, udf):
+    def applyInPandas(self, func, schema):
         """
-        Applies a function to each cogroup using a pandas udf and returns the result
+        Applies a function to each cogroup using pandas and returns the result
         as a `DataFrame`.
 
-        The user-defined function should take two `pandas.DataFrame` and return another
+        The function should take two `pandas.DataFrame` and return another
```

Comment:
minor but should be plural -> `pandas.DataFrame`s. I'm not sure the correct format though.. I see `pandas.DataFrame`\\s and ``DataFrame``s below